dashboards/MySQL_Replication.json

```diff
@@ -720,6 +810,7 @@
             "dashLength": 10,
             "dashes": false,
             "datasource": "Prometheus",
+            "description": "Shows the amount of data written to the binlog files during the last 24 hours and grouped into hourly buckets. This graph can give you an idea of the amount of changes to your data.",
```

Comment:
IMHO the notice about 24 hours has to be removed.
It's true only if quick range - "Last 24 hours" is used.
![screenshot_20181114_100203](https://user-images.githubusercontent.com/25184101/48468625-129bbc00-e7f5-11e8-8603-5f0ae015c16f.png)